string node_name
string part_name
uint64 timestamp
